app/controllers/profiles_controller.rb

```diff
@@ -116,7 +114,7 @@ def find_car
   end
 
   def user_params
-    params.require(:user).permit(:photo, :id_document, :first_name, :last_name, :email, :phone_number, :description, :gender, :facebook_URL, :linkedin_URL, :address, :date_of_birth)
+    params.require(:user).permit(:photo, :id_document, :first_name, :last_name, :email, :phone_number, :description, :gender, :facebook_url, :linkedin_url, :address, :date_of_birth)
   end
```

Comment:
not sure if params are case-insensitive. I recommend downcase as rubocops does, but in that case ensure that this change is applied everywhere